src/asciidoc/docs/blueprints/customVCC.adoc

```diff
@@ -0,0 +1,208 @@
+= Custom VCC Template
+:page-slug: custom-vcc-blueprint
+
+Follow this blueprint to build a template for a custom Koji VCC to give remixes the option to select from a drop down of choices which are retrieved from a third party API.
```

Comment:
```suggestion
Follow this blueprint to build a custom VCC that allows remixers to select from a drop-down list of cat breeds.
You will retrieve the list from a third-party API and style the drop-down to look like a native Koji control.
```

Also, I'd like to create a new section on the Blueprints Overview page called "Build your custom VCC" and add the "Cat Selector VCC" as the first item (rather than putting it with the "templates"). For some reason, I don't see that file in the commit so maybe it's not bundled with this pull request or I'm looking in the wrong place.